src/main/scala/io/mesosphere/mesosaurus/Mesosaurus.scala

```diff
@@ -175,7 +186,8 @@ object Mesosaurus extends Logging {
                 throw new ArgumentParserException("mem standard deviation must be > 0", parser)
             }
             val port = getInt(options, PORT, DEFAULT_PORT)
-            return (master, failover, port, new TaskGenerator(tasks, duration, durationSigma, arrival, load, cpus, cpusSigma, mem, memSigma))
+            return (master, failover, port, new TaskGenerator(tasks, duration,
+                durationSigma, arrival, load, cpus, cpusSigma, mem, memSigma, failRate = fail))
```

Comment:
``` scala
TaskGenerator(requestedTasks: Int,
        taskDurationMean: Int,
        taskDurationSigma: Int,
        arrivalTimeMean: Int,
        load: Double,
        cpusMean: Double,
        cpusSigma: Double,
        memMean: Long,
        memSigma: Long,
        offerAttempts: Int = 100,
        failRate: Double = 0.0)
```
To avoid re-arranging the argument order for TaskGenerator. Is there an easier way to do this? Or should I just re-arrange them so this isn't needed? Could also make them all explicitly pass-by-name.